.eslintrc.js

```diff
@@ -40,6 +41,15 @@ module.exports = {
         '@typescript-eslint/no-invalid-this': 'off',
         '@typescript-eslint/no-magic-numbers': 'off',
         '@typescript-eslint/no-parameter-properties': 'off',
+        //had to add this rule to prevent eslint from crashing
+        '@typescript-eslint/no-restricted-imports': ['off', {}],
+        //mitigating this sometimes results in undesirably verbose code. Should investigate enabling again in the future.
+        '@typescript-eslint/no-unsafe-argument': 'off',
```

Comment:
In theory, I like preventing `any` from being in the code. But there are a lot of changes required to enable this rule, and overall, most of the changes were a little too verbose for certain situations that don't really need it. So disabling the rule for now, but we can revisit in the future if if we want to tighten up the `any` rules.

Reply:
Make sense to open a GH issue with a good first issue label or help wanted label?

Reply:
Sure! #457